.travis.yml

```diff
@@ -0,0 +1,30 @@
+language: cpp
+compiler: gcc
+sudo: require
+dist: trusty
+
+script:
+  - ./autogen.sh # Undocumented
```

Comment:
It's not really undocumented. The top-level README.md mentions this as a step when building from a pristine checkout.

Reply:
I was referring to https://github.com/qpdf/qpdf/blob/master/INSTALL, maybe this information could be added there too. Will remove my comment